src/test/java/persistence/sql/ddl/CreateQueryBuilderTest.java

```diff
@@ -35,5 +37,21 @@ void testGenerateQuery_WhenPersonEntity_ThenGenerateDdl() {
             //then
             assertThat(ddl).isEqualTo("CREATE TABLE " + table.getTableName()+ " (id BIGINT AUTO_INCREMENT PRIMARY KEY,nick_name VARCHAR,old INTEGER,email VARCHAR NOT NULL)");
         }
+
+        @DisplayName("Order Entity와 OrderItem Entity 테이블 생성 ddl이 만들어지는지 확인한다.")
+        @Test
+        void testGenerateQuery_WhenOrderEntity_ThenGenerateDdl() {
+            // given
+            Table table = Table.getInstance(Order.class);
+            Table table1 = Table.getInstance(OrderItem.class);
+
+            //when
+            String ddl = builder.generateQuery(table);
+            String ddl2 = builder.generateQuery(table1);
+
+            //then
+            assertThat(ddl).isEqualTo("CREATE TABLE " + table.getTableName()+ " (id BIGINT AUTO_INCREMENT PRIMARY KEY,order_number VARCHAR)");
```

Comment:
```suggestion
    String expected = "CREATE TABLE orders (id BIGINT AUTO_INCREMENT PRIMARY KEY,order_number VARCHAR)";
            assertThat(ddl).isEqualTo(expected);
```

테스트는 검증하고자하는 행동이 분명하고 1000번 돌려도 일정한 값이 나와야 한다고 생각해요. 동적으로 맵핑 할 이유는 저는 없다고 생각해요. 🤔 

동적 맵핑, 상수 맵핑 각각의 장단점이 있을것 같은데 이부분에 대해서 어떻게 생각하시나요?

Reply:
고정된 값으로 테스트 하는게 좋아 보입니다.
수정하도록 하겠습니다.